mmf/configs/datasets/vqa2/for_butd_caption_generation.yaml

```diff
@@ -0,0 +1,36 @@
+dataset_config:
```

Comment:
I don't think caption generation for VQA is required. VQA uses COCO images only.